rl_loop/train_and_validate.py

```diff
@@ -112,7 +120,7 @@ def loop(unused_argv):
             continue
         with utils.timer("Validate"):
             validate_pro()
-            validate_holdout_selfplay()
+            #validate_holdout_selfplay()
```

Comment:
intended?

Reply:
yep -- the holdout isn't working right now, i'll come back to it.